python/analytics/bottle.py

```diff
@@ -34,62 +31,47 @@
 import tempfile
 import threading
 import time
+import types
+import imp
 import warnings
-
-from six.moves.http_cookies import SimpleCookie
+import _thread
+import http.client as http_client
+from http.cookies import SimpleCookie
 from tempfile import TemporaryFile
 from traceback import format_exc
 from urllib import parse
+from io import BytesIO
 
-try: from collections import MutableMapping as DictMixin
-except ImportError: # pragma: no cover
+try:from collections.abc import MutableMapping as DictMixin
```

Comment:
Can we fix the formatting here and elsewhere in the PR to indent and newline?

Reply:
we have updated the formatting for this and other examples.